packages/discovery-provider/integration_tests/challenges/test_connect_verified_challenge.py

```diff
@@ -40,7 +40,7 @@ def test_connect_verified_challenge(app):
 
     with db.scoped_session() as session:
         bus = ChallengeEventBus(redis_conn)
-        session.query(Challenge).filter(Challenge.id == "connect-verified").update(
+        session.query(Challenge).filter(Challenge.id == "v").update(
```

Comment:
Similar suggestion to changes in the other PR: Using some constants here to make the code more readable would be appreciated.